modules/bazel_gomock/0.1.0-1/MODULE.bazel

```diff
@@ -0,0 +1,18 @@
+module(
+    name = "bazel_gomock",
+    version = "0.1.0-1",
```

Comment:
According to the semver spec, this version would be considered a prerelease to 0.1.0 and thus sort before it. The usual pattern we recommend is `0.1.0.bcr.1`, which relies on the support for extended semver and sorts correctly. Do you want to switch to that?